packages/test-apps/simple-react/server.ts

```diff
@@ -110,7 +110,7 @@ export const server = async () => {
 if (require.main === module) {
   void server().then((server) => {
     console.warn(server.printRoutes())
-    return server.listen(3000).then((address) => {
+    return server.listen({ port: 3000, host: '0.0.0.0' }).then((address) => {
```

Comment:
It's better this way IMO. 127.0.0.1 makes it annoying to work with in codespaces.